adapters/httpapi_compact.c

```diff
@@ -744,6 +746,12 @@ static HTTPAPI_RESULT OpenXIOConnection(HTTP_HANDLE_DATA* http_instance)
             result = HTTPAPI_SET_OPTION_FAILED;
             LogInfo("Could not load the client certificate private key");
         }
+        else if ((http_instance->tls_renegotiation != 0) &&
```

Comment:
Shall check `tls_renegotiation` for `bool`, not `int`. Please see comment below.

Reply:
Fixed on the commit 98666e9b4d46e63d900d4abd1bc1591d3049b5a6